src/Sylius/Bundle/AdminBundle/Resources/views/Dashboard/_chart.html.twig

```diff
@@ -1,12 +0,0 @@
-{% set labels, values = sales_summary.intervals, sales_summary.sales %}
```

Comment:
Removing a template is a BC Break (https://docs.sylius.com/en/latest/book/organization/backward-compatibility-promise.html#templates)
You should add sth like at the begging of this template:
`{% deprecated "This template is deprecated since Sylius 1.8" %}`
and do not remove the file

Reply:
got it back